s3-s3-replication-sam/example-pattern.json

```diff
@@ -0,0 +1,67 @@
+{
+  "title": "S3 to S3 replication",
+  "description": "Create a stack template that you can use to replicate objects between S3 buckets across AWS accounts and regions.",
+  "language": "Python",
+  "level": "200",
+  "framework": "SAM",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
```

Comment:
Please keep this shorter, especially because the pattern does not set up cross-region or cross-account replication

Reply:
yes, thanks

Reply:
Hi, this section is still very verbose. Please keep it a bit shorter and focus on _"how it works"_. Something like: _"This pattern sets up S3 replication between..."_

Reply:
Ok updated :)